### PR TITLE
DEV BRANCH - Fixed/added React Discord Components

### PR DESCRIPTION
**Pull request by KennySB-dev**
Fixed/added the Discord Components, and fixed some stuff in the create.mdx file

The project should be linked once its pulled to the master branch imo :D

### CODE REVIEW

File: pages/server/create.mdx [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-We have over **24** different server types you can choose from to start your hosting experience!
+We have over **49** different server types you can choose from to start your hosting experience!
 
 <Tip>
   For the purpose of this page we will just be showing the create server list,
   you can look at the other guides to see how to create and work with them
 </Tip>
 
 (For non-donators)
-By running `DBH!server create list` you can see the list of all the servers you can create!
```
Comment (Kyro3400) [inline pages/server/create.mdx:11]: The command is actually `DBH!server list` no longer `DBH!server create` as the way it was wrote is that the list command shows everything that the images show.

File: pages/server/create.mdx [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+By running `DBH!server create` you can see the list of all the servers you can create!
 
 ![Server Creation List (For Non Donator Members)](/content/create-server/normal-server-create-list.png)
 
 (For donators)
-By running `DBH!server create-donator list` you can see the list of all the servers you can create!
```
Comment (Kyro3400) [inline pages/server/create.mdx:16]: Same with the one above.

File: pages/server/development/aio.mdx [hunk 3]
```diff
@@ -18,85 +20,110 @@ In that case, go to DBH server and run this command:
...
-          126{" "}
-        </DiscordEmbedField>
-        <DiscordEmbedField fieldTitle="Server name">
-          {" "}
-          Untitled Server (settings -> server name){" "}
-        </DiscordEmbedField>
-        <DiscordEmbedField fieldTitle="Type"> aio </DiscordEmbedField>
-        <DiscordEmbedField fieldTitle="WARNING">
-          {" "}
-          Please do not host game servers on java or AIO servers. If you need a
```
Comment (Kyro3400) [inline pages/server/development/aio.mdx:92]: This line is still needed for both donator and non donator.

File: pages/server/development/rdb.mdx [hunk 4]
```diff
@@ -1,88 +1,129 @@
...
 
 Red discord bot is a multi moduler discord bot. It means that it just has some base of commands. And you can add more commands by adding a thing called cogs (we can say plugins in mc fashion 😎) it had over 550 cogs and has many commands by default.
 
-# Creating a server
+# Creating the server
 
-To create a server, go to DBH's discord server and type the commands below
+To create a server, we suppose that you've already [created and linked an account into the panel](/getting-started).
+
+In that case, go to DBH server and run this command:
```
Comment (Kyro3400) [inline pages/server/development/rdb.mdx:19]: This should be grammatically correct, which is how it was before `go to DBH’s Discord server and run this command:`
Or `go to the DBH Discord server and run this command:`

File: pages/server/software/gitea.mdx [hunk 5]
```diff
@@ -11,17 +21,120 @@ Apart from a Git with a cup of Tea:
 
 ---
 
-We ask you to read entirely this document because if you skip one step you might break the server and you will need to reinstall everything.
+It is important that you thoroughly read and follow this document, as one mistake might end up in you having to re-make the server.
 
 # Creating the server
 
-To create a server, we suppose that you've already [created and linked an account into the panel.](/getting-started)
```
Comment (Kyro3400) [inline pages/server/software/gitea.mdx:18]: This really isn’t needed to be change, the period can stay there.

File: pages/server/create.mdx (hunk 1 above)
Comment (KennySB-dev) [reply pages/server/create.mdx:11]: ah ok thank you

Comment (KennySB-dev) [reply pages/server/create.mdx:11]: wait but server list shows the person's servers, when the images show the server list?

File: pages/server/software/gitea.mdx (hunk 5 above)
Comment (KennySB-dev) [reply pages/server/software/gitea.mdx:18]: oh i changed the entire code by copying and pasting them so theyre all similiar
i can add it back if you want

File: pages/server/development/aio.mdx (hunk 3 above)
Comment (KennySB-dev) [reply pages/server/development/aio.mdx:92]: fixing now

File: pages/server/create.mdx (hunk 1 above)
Comment (Kyro3400) [reply pages/server/create.mdx:11]: Hmm guess that whole command got changed, guess it is just create

**Review by Kyro3400 — APPROVED** [review]

**Pull request merged**